nodejs/webapi/swagger/openapiservice.js

```diff
@@ -0,0 +1,58 @@
+const swaggerJsdoc = require('swagger-jsdoc');
+const swaggerUi = require('swagger-ui-express');
+
+// Basic swagger definition
+const BasicswaggerDefinition = {
+    definition: {
+        openapi: "3.0.0",
+        info: {
+         title: "ApiApp",
```

Comment:
Would you be able to change `ApiApp` to something else like `webapi` that represents your api app?